scripts/pkg/build_templates/opensearch/rpm/opensearch.rpm.spec

```diff
@@ -175,16 +169,27 @@ exit 0
 %attr(0644, root, root) %config(noreplace) %{_prefix}/lib/sysctl.d/%{name}.conf
 %attr(0644, root, root) %config(noreplace) %{_prefix}/lib/tmpfiles.d/%{name}.conf
 
+%dir %attr(750, %{name}, %{name}) %{data_dir}
+%dir %attr(750, %{name}, %{name}) %{log_dir}
+%dir %attr(750, %{name}, %{name}) %{pid_dir}
```

Comment:
I think I initially put log and pid dir without %dir is to prevent them to be cleaned up by rpm:
```
Identifies that the path is a directory that should be owned by this RPM. This is important so that the RPM file manifest accurately knows what directories to clean up on uninstall. 
```

I dont remember why I dont do the same for data.

Reply:
We need to dig into this: I am not used to rpm and with or without `%dir` I see the same behavior: directory is not removed if new files where added into it.

We indeed do not want to remove the data dir when uninstalling.

Reply:
Done!